### PR TITLE
Delegation

### PR DESCRIPTION
**Pull request by xenide**
1. Delegation works, to oneself or to another address
2. queueProposal call implemented
3. Added mainnet governance addresses and modified testnet governance addresses 

Preview deployment link is below. Contact Alex if you need testnet tokens for testing

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -91,6 +103,40 @@ export default function Home() {
...
+  /**
+   * Calls the delegate function with the delegate
+   * being the given input address
+   * @param {event} event context from which this is fired
+   */
+  const handleDelegate = async (event) => {
+    let delegateAddress = null;
+
+    // Get the delegate's address
+    if (event.target.innerText === 'Delegate') {
```
Comment (andersonaguiar) [inline pages/index.js:115]: maybe a switch block to avoid repetition?

**Review by OliverNChalk — APPROVED** [review]
Again, not super qualified but it looks good to me. Would be keen for Anderson to take a look pre-merging

**Review by leodsoft — APPROVED** [review]

**Review by KennethAshley — APPROVED** [review]
This looks awesome!

File: state/governance.js [hunk 2]
```diff
@@ -278,6 +297,42 @@ function useGovernance() {
...
+
+    // Wait for tx to be confirmed and mined
+    while(!txReceipt) {
+      await ticker.next();
+      txReceipt = await txVisitor.getReceipt();
+      console.log("txReceipt:", txReceipt);
+    }
+
+    // Handle failed tx
+    if (txReceipt.reverted) {
```
Comment (andersonaguiar) [inline state/governance.js:327]: is there a way to simulate this fail? and handle it?

File: pages/index.js (hunk 1 above)
Comment (xenide) [reply pages/index.js:115]: we only have 2 possible cases here, 3 if you include error handling. Does it still make sense to use a switch block?

**Review by andersonaguiar — APPROVED** [review]

File: state/governance.js (hunk 2 above)
Comment (xenide) [reply state/governance.js:327]: for queuing it's harder to fail. But right now I don't have a reliable way to simulate the failure.

For handling failure conditions (for all smart contract calls) there's a card (task) for it here 
https://trello.com/c/bJtC361w/23-additional-ui-handling-when-proposal-creation-transaction-fails

**Pull request merged**